code/src/App.js

```diff
@@ -1,9 +1,27 @@
 import React from 'react'
+import { BrowserRouter, Route, Switch, Redirect } from 'react-router-dom';
+
+import MovieList from 'components/MovieList';
+import MovieDetails from 'pages/MovieDetails';
+import NotFound from 'pages/NotFound';
 
 export const App = () => {
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
+    <BrowserRouter>
+      <Switch>
+        <Route path="/" exact>
+            <MovieList />
+        </Route>
+        <Route path="/movie/:movieId" exact>
+          <MovieDetails />
+        </Route>
+        <Route>
+          <NotFound />
+        </Route>
+        <Redirect 
```

Comment:
Really cool that you added a Redirect-page! 👍 Unfortunately I can't see it in the deployed page, there's only the customized one from Netlify showing. There's a Slack-Thread about this issue: https://technigo.slack.com/archives/C6QAN25EV/p1604503267142900